ReactAndroid/src/main/java/com/facebook/react/modules/camera/CameraRollManager.java

```diff
@@ -294,7 +314,7 @@ protected void doInBackgroundGuarded(Void... params) {
       // an SQLite DB and forwards parameters to it without doing any parsing / validation.
       try {
         Cursor photos = resolver.query(
-            Images.Media.EXTERNAL_CONTENT_URI,
+            MediaStore.Files.getContentUri("external"),
```

Comment:
Is there an Android constant for this? Is this the same as `Environment.getExternalStorageDirectory()`?

http://stackoverflow.com/questions/17215466/environment-getexternalstoragedirectory-getabsolutepath-gives-a-different-pa

Prefer using a constant.